discord/commands/options.py

```diff
@@ -390,6 +387,47 @@ def to_dict(self) -> dict:
     def __repr__(self):
         return f"<discord.commands.{self.__class__.__name__} name={self.name}>"
 
+    @property
+    def autocomplete(self):
+        return self._autocomplete
+
+    @autocomplete.setter
+    def autocomplete(self, value) -> None:
```

Comment:
Are property setters docstrings showed...?

Reply:
I mistakenly added the docstring to the setter instead of the getter mb